deepchecks/utils/distribution/drift.py

```diff
@@ -274,6 +282,9 @@ def calc_drift_and_plot(train_column: pd.Series,
     categorical_drift_method: str, default: "cramer_v"
         decides which method to use on categorical variables. Possible values are:
         "cramer_v" for Cramer's V, "PSI" for Population Stability Index (PSI).
+    relative_change: bool, default: False
+        If True, the drift score will be use the relative change between the two distributions, disregarding the
+        absolute values. In this case, the plot will be created with a log scale in the y-axis.
```

Comment:
Why log scale though? Can you show how it looks like?

Reply:
![image](https://user-images.githubusercontent.com/17730502/216331293-884f4108-6026-42da-8968-7171a3436575.png)